scripts/analyse.sh

```diff
@@ -1,37 +1,93 @@
-#!/bin/sh
+#!/usr/bin/env bash
+# shellcheck disable=SC2016,SC2207,SC2002
 
-set -ex
+set -e
 
-BP=$(dirname $0)
+# generator_log_prefix="${1:-generator/tx-gen}"
+# explorer_log_prefix="${2:-node-on-explorer/node}"
+generator_log_prefix="${1:-logs/generator}"
+explorer_log_prefix="${2:-logs/node}"
 
-# extract txs from generator
-${BP}/xtx.sh generator/tx-gen-*.json > txs
-sort -k 1 -t ';' txs > txs1
+mkdir -p 'analysis'
+cd 'analysis'
+rm -f ./*
+BP=../$(dirname "$0")
 
-# extract txs in blocks received
-rm -f bks
-for L in node-on-explorer/node-*.json; do
-  ${BP}/xbk.sh $L >> bks
-done
+sender_logs=($(ls ../"${generator_log_prefix}"-*.json))
+receiver_logs=($(ls ../"${explorer_log_prefix}"-*.json))
 
-sort -k 2 -t ';' bks > bks1
+###
+### Since we're using intermediate files with intricate semantics of sorting,
+### we better use more descriptive file names, let we go awry..
+###
```

Comment:
good idea! is certainly helpful

Reply:
is the naming schema also valid on other platforms? I could sense that Windows will not be happy with a ";" in between. So a simple "_" would be safer

Reply:
Good point about Windows, that certainly should break! Fixed.